src/scripts/search.js

```diff
@@ -0,0 +1,135 @@
+import * as pagefind from '/pagefind/pagefind.js';
+
+const url = new URLSearchParams(window.location.search);
+
+pagefind.options({
+  baseUrl: '/',
+});
+
+const searchTerm = url.get('q');
+let index = url.get('page');
+let lang = 'en';
+
+if (window.location.href.includes('/fr/')) {
+  lang = 'fr';
+}
+
+if (!index) {
+  index = 1;
+}
+
+if (searchTerm) {
+  // I realized we don't have a value property for gcds-search, want to add that
+  // document.getElementById('searchbar').value = searchTerm;
+  const search = await pagefind.search(searchTerm);
```

Comment:
Should we add any error handling here in case of network failures or invalid responses?